fake-vm-component-scripts/fake-vm/launch.py

```diff
@@ -10,25 +10,30 @@
 
 instances = []
 
-def fix_config_keys(source):
-    for k, v in source:
-        yield (k, v) if '.' not in k else (k.split('.')[1], v)
+config_keys = {
+    k: k.split('.')[1] for k in {
+        'configuration.name',
+        'configuration.address',
+        'configuration.login',
+        'configuration.failure',
```

Comment:
For `launch.py` name is useless directly, but it is needed for discovery because discovery sets the new instance name to this value. `failure` is used to emulate failures - see `healthCheck` script.